src/components/earn/UnstakingModal.tsx

```diff
@@ -90,19 +90,19 @@ export default function UnstakingModal({ isOpen, onDismiss, stakingInfo }: Staki
           </RowBetween>
           {stakingInfo?.stakedAmount && (
             <AutoColumn justify="center" gap="md">
-              <ThemedText.DeprecatedBody fontWeight={600} fontSize={36}>
+              <ThemedText.HeadlineLarge>
```

Comment:
After adding `HeadlineLarge` from Figma, I did a quick search to replace places where `DeprecatedBody` was used to achieve same settings